CHANGELOG.md

```diff
@@ -1,3 +1,50 @@
+## 2.3.0 / 2023-05-18
+### Features
+- Feat: Adding influxdb plugin (#690)
+- Feat: Fluentd in_tail plugin (#753)
+- Feat: add AWS Kinesis Data Streams output plugin for Fluent Bit (#768)
+- Feat:
+
+### ENHANCEMENT
+- Add EnvVars support to FluentD (#697)
```

Comment:
Consider moving the following item with `add xxx` to features section:

```markdown
- Add EnvVars support to FluentD (#697)
- Add Pod Annotations support to FluentD (#698)
- Fluent operator & fluentbit: Added tolerations, nodeSelector + more (#704)
- Add fluentbit.affinity configuration (#726)
- Allow setting fluentd PodSecurityContext (#744)
- Add missing fluentd buffer options (#757)
- Add global log_level support for fluentd (#770)
- Add scheduler support for fluentbit & fluentd (#771)
```

Reply:
fixed